Source/Mod/Core/ModAssemblyLoadContext.cs

```diff
@@ -0,0 +1,298 @@
+using System.Collections.Concurrent;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Runtime.Loader;
+using Mono.Cecil;
+
+namespace Celeste64.Mod;
+
+/// <summary>
+/// A mod's assembly context, which handles resolving/loading mod assemblies
```

Comment:
Super minor suggestion. If I understand correctly, I think this was based partially on what everest does. It might be nice to give credit or give a link back to everest's version as a comment so people can find the original for reference in the future. Although, if you feel that it is different enough, it's probably not that big of a deal. Not a requirement, but just a suggestion.

Reply:
Can do that